### PR TITLE
Added albums service

### PR DESCRIPTION
**Pull request by ranuccimariano**
## Summary

Added a service to list the albums and the photos from a specific album from the JSONPlaceHolder API.

## Trello Card

https://trello.com/c/GUkpYgPn

### CODE REVIEW

File: app/services/albums.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const rp = require('request-promise');
+
+module.exports.listAlbums = () => {
+  rp('https://jsonplaceholder.typicode.com/albums')
```
Comment (molinajulian) [inline app/services/albums.js]: Add environment var for this url

File: app/services/albums.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    .then(response => console.log(response))
```
Comment (molinajulian) [inline app/services/albums.js]: `.then` is not necessary

File: app/services/albums.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    .catch(err => console.log(err.message));
```
Comment (molinajulian) [inline app/services/albums.js]: Use `logger.error` and throw a known error

File: app/services/albums.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+};
+
+module.exports.listAlbumPhotos = album_id => {
```
Comment (molinajulian) [inline app/services/albums.js]: In general we use camel case. `albumId`

File: app/services/albums.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  rp(`https://jsonplaceholder.typicode.com/photos?albumId=${album_id}`)
```
Comment (molinajulian) [inline app/services/albums.js]: Apply the same comments in this function

File: app/services/albums.js (hunk 1 above)
Comment (ranuccimariano) [reply app/services/albums.js]: Done :+1:

File: app/services/albums.js (hunk 3 above)
Comment (ranuccimariano) [reply app/services/albums.js]: Done :+1:

File: app/services/albums.js (hunk 4 above)
Comment (ranuccimariano) [reply app/services/albums.js]: Done :+1:

File: app/services/albums.js (hunk 5 above)
Comment (ranuccimariano) [reply app/services/albums.js]: Done :+1:

File: app/services/albums.js (hunk 2 above)
Comment (ranuccimariano) [reply app/services/albums.js]: This will be applied in the next pull request.

File: app/services/albums.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
+const rp = require('request-promise');
+const util = require('util');
```
Comment (molinajulian) [inline app/services/albums.js]: Add empty line here

**Pull request merged**